options/options.go

```diff
@@ -91,6 +91,7 @@ func New() (*Options, error) {
 	}
 
 	// TODO(options): Move this set-or-default logic to its own function.
+	opts.ScorecardOpts.Format = os.Getenv(EnvInputResultsFormat)
```

Comment:
The struct fields for [ScorecardOpts](https://github.com/ossf/scorecard/blob/4b9f0389c6f6/options/options.go#L30) are named differently (ex. "SCORECARD_RESULTS_FORMAT" instead of "INPUT_RESULTS_FORMAT"), so env.Parse won't work here. Is there a workaround?

Reply:
I think it's fine to update the names to `INPUT_RESULTS_FORMAT`

Reply:
Should we update `options/env.go` to contain a struct instead of a list of `consts`? So basically,

```
// options/env.go

type Env struct {
  EnableSarif `env:ENABLE_SARIF`
  AuthToken `env:INPUT_REPO_TOKEN`
 ...
}

// options/options.go

env.Parse(Env)
```

Reply:
> I think it's fine to update the names to `INPUT_RESULTS_FORMAT`

I don't think I can edit that file since its importing from a previous commit.

> Should we update `options/env.go` to contain a struct instead of a list of `consts`? So basically,

Yeah this would make the code a lot more clean. Right now there are a lot of duplicate vars for the env vars.

Reply:
I updated options/env.go to be a struct so it reduced the amount of OS calls in options. Also tested and verified that the workflow runs properly.